core/domain/rights_manager.py

```diff
@@ -567,6 +567,8 @@ def check_can_voiceover_activity(user, activity_rights):
 
 def check_can_manage_voice_artist_in_activity(user, activity_rights):
```

Comment:
Does it still make sense to have this function or should we split it into add/remove?

Reply:
I think so, the main reason is that we want to have a different error for adding voice artists to private exploration and splitting this into two functions would make it to be almost the same except for the private/published check, but the private/published check is not in different places anyway.